pyOCD/tools/pyocd.py

```diff
@@ -22,8 +22,9 @@
 
 import os
 import pyOCD
-from pyOCD.board import MbedBoard
-from pyOCD.target import target_kinetis
+from ..board import MbedBoard
+from ..target import target_kinetis
+from .. import __version__
```

Comment:
You should just use absolute imports here.  The base namespace of `pyOCD` is going to be more stable than the relative position of this file to the base namespace.  See https://www.python.org/dev/peps/pep-0008/#imports